packages/mdast-util-split/README.md

```diff
@@ -0,0 +1,496 @@
+# mdast-util-split [![Build Status][build-badge]][build-status] [![Coverage Status][coverage-badge]][coverage-status]
+
+
+**rebber** is a LaTeX stringifier for [remark][]
+
+## Installation
+
+[npm][]:
+
+```bash
+npm install mdast-util-split
+```
+
+## Usage
+
+```javascript
+const unified = require('unified')
+const parse = require('remark-parse')
+const split = require('mdast-util-split')
+
+var tree = unified()
+  .use(parse)
+  .parse('# part\n\n## chapter \n\n Hello world \n\n # part *2*')
+
+console.log(split(tree))
+```
+
+## API
+
+### `split(node[, options])`
+
+Splits a MDAST tree into separate trees by [heading depth](https://github.com/syntax-tree/mdast#heading).
+
+#### `options.splitDepth = 1`
+
+An integer greater or equal to 1 determining the max heading depth at which the tree is split.
+
+#### `options.introductionAsProperty = true`
+
+A boolean driving the way we want to process introduction :
+- if `true`, all MDAST elements between the split header and the first sub header will be extracted and put in the `introduction` property of the result
+property of the returned object
+- if `false` they will just be added as MDAST elements of the `children` property.
```

Comment:
I don't understand this line, could you please clarify?

Reply:
if the property set to false, we do not extract the introduction from the tree. In though I had changed like for the conclusion in `if false, the part tree will not be changed`